pettingzoo/butterfly/cooperative_pong/cooperative_pong.py

```diff
@@ -217,6 +217,8 @@ def __init__(self, randomizer, ball_speed=9, left_paddle_speed=12, right_paddle_
         original_color_shape = (original_shape[0], original_shape[1], 3)
         # self.observation_space = [gym.spaces.Box(low=0.0, high=1.0, shape=(original_shape), dtype=np.float32) for _ in range(self.num_agents)]
         self.observation_space = [gym.spaces.Box(low=0, high=255, shape=(original_color_shape), dtype=np.uint8) for _ in range(self.num_agents)]
+        # define the global space of the environment or state
+        self.state_space = gym.spaces.Box(low=0, high=255, shape=((original_shape[0], original_shape[1] * 2, 3)), dtype=np.uint8)
```

Comment:
The dimensions of this space can be cleaner. 
shape=(self.s_height, self.s_width, 3)